content/docs/0.6.0/reference/monitoring/prometheus/index.md

```diff
@@ -68,6 +68,17 @@ create a *secret* containing the **user**, **password**, and **url**. The secret
 
 To tell the *prometheus-sli-service* how to acquire the values of an SLI, the correct query needs to be configured. This is done by adding an SLI configuration to a project, stage, or service using the [add-resource](../../cli/#keptn-add-resource) command. The resource identifier must be `prometheus/sli.yaml`.
 
+Please take a look at this snippet which implements a concrete SLI file to learn more about the structure of a SLI file. It is possible to use placeholders such as `$PROJECT`, `$SERVICE`, `$STAGE` and `$DURATION_SECONDS` in the queries.
+
+```yaml
+---
+spec_version: '1.0'
+indicators:
+  response_time_p50: histogram_quantile(0.5, sum by(le) (rate(http_response_time_milliseconds_bucket{handler="ItemsController.addToCart",job="$SERVICE-$PROJECT-$STAGE"}[$DURATION_SECONDS])))
```

Comment:
The SLIs provided here are very specific to sockshop (`handler="ItemsController.addToCart"`). I wonder if we should remove that piece?

Reply:
sure, I can remove it

Reply:
removed it